utils/chem/chem_diagb.h

```diff
@@ -230,13 +230,38 @@ namespace gdasapp {
         }
       }
 
-      // Rescale
-      if (fullConfig.has("rescale")) {
-        double rescale;
-        fullConfig.get("rescale", rescale);
-        util::multiplyFieldSet(bkgErrFs, rescale);
+     // Rescale
+      if (fullConfig.has("global rescale")) {
+        const eckit::LocalConfiguration GlobalRescaleConfig(fullConfig, "global rescale");
+        const eckit::LocalConfiguration GlobalRescaleGeomConfig(GlobalRescaleConfig, "geometry");
+        const fv3jedi::Geometry GlobalRescaleGeom(GlobalRescaleGeomConfig, this-> getComm());
+        fv3jedi::Increment global_rescale(GlobalRescaleGeom, chemVars, cycleDate);
+        global_rescale.zero();
+        const eckit::LocalConfiguration GlobalRescaleStdConfig(GlobalRescaleConfig,
+                                                    "rescale stddev");
+        // Get the 'datapath' and 'filename_trcr' from the YAML configuration
+        std::string datapath, filename_trcr;
+        GlobalRescaleStdConfig.get("datapath", datapath);
+        GlobalRescaleStdConfig.get("filename_trcr", filename_trcr);
```

Comment:
making a note, that if we were to use 'histories' and not 'restarts' this would cause it to fail. Do we need the filename printed out?

Reply:
Sure. Like 

oops::Log::info() <<"filename_trcr", filename_trcr << std::endl;     ?

Reply:
my point was that if you use a different IO class, `filename_trcr` won't exist, so it will fail.